### PR TITLE
Chores/movie quotes generator

### PR DESCRIPTION
**Pull request by dibashthapa**

### CODE REVIEW

**Review by TheLearneer — CHANGES_REQUESTED** [review]
Please remove the changes that are not related to this PR

**Comment by dibashthapa** [comment]
> Please remove the changes that are not related to this PR

Done @TheLearneer

**Review by TheLearneer — CHANGES_REQUESTED** [review]
Please go through the requested changes 🙇

**Comment by dibashthapa** [comment]
@TheLearneer Made changes accordingly

File: src/content/apis/entertainment/movie-quotes.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: Web Series Quotes Api
+description: Quotes of various web series with image generating feature
+
```
Comment (TheLearneer) [inline src/content/apis/entertainment/movie-quotes.md:4]:
```suggestion
```

Comment (dibashthapa) [reply src/content/apis/entertainment/movie-quotes.md]: Done @TheLearneer

**Review by TheLearneer — APPROVED** [review]
LGTM, thank you for your contribution @dibashthapa

**Pull request merged**